tests/unit/project_api/canned_responses.py

```diff
@@ -0,0 +1,176 @@
+class Projects(object):
```

Comment:
Nit: I'm ok with this, but it would be more pythonic to use separate modules rather than classes for the namespaces of the canned responses.